client/src/components/Forms/AuthForm/AuthHeader/AuthHeader.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from 'components/Forms/AuthForm/AuthHeader/AuthHeader.module.scss';
+
+const AuthHeader = ({ hasAccount }) => {
+  return (
+    <div className={styles.authHeader}>
+      <h1 className={styles.title}>Instaglam</h1>
+      {hasAccount ? (
+        <h2 className={styles.h2}>
+          Sign up to see photos and videos from your friends.
+        </h2>
+      ) : null}
+    </div>
+  );
+};
```

Comment:
You don't need the return keyword here also

const AuthHeader = ({ hasAccount }) => (
    <div className={styles.authHeader}>
      <h1 className={styles.title}>Instaglam</h1>
      {hasAccount ? (
        <h2 className={styles.h2}>
          Sign up to see photos and videos from your friends.
        </h2>
      ) : null}
    </div>
  );